app.js

```diff
@@ -27,6 +30,7 @@ app.use((err, req, res, next) => {
 });
 
 app.use((err, req, res, next) => {
+  console.log(err);
```

Comment:
You might want to still have an app.all error handler to make sure you can handle 404 errors that are in the routing and outwith the scope of your controllers and models